dist/codecov_ats.sh

```diff
@@ -114,28 +114,30 @@ if [[ -z $response && -n $INPUTS_OVERRIDE_BASE_COMMIT ]]; then
 	exit 1
 fi
 
-response=$(echo $response | sed 's/,//g')
-runner_options=$(echo $response | sed 's/^.*runner_options\": \[//' | sed 's/\].*$//')
```

Comment:
@giovanni-guidini can we not just pipe this result to a file instead? that avoids (1) the `jq` use and (2) allows us to save as `.txt` instead of `.json`

Reply:
Yes, but I'm not convinced this code can properly parse all test names. I think `jq` is better because it can take advantage of the structured nature of `json` instead of txt.

As for saving the result as .txt instead of .json, I can do that if preferred. But I don't really see why it would be preferred, if we also suggest a reliable way to parse the result (see README.md). Once again, the fact that .json is more structured might be beneficial

Reply:
ok, I can include `.txt` files too.

> As for jq, I'm ok with it, but I think we're going to be shooting ourselves in the foot later when we have to support alpine.

You are probably correct here. But I hope to solve this issue by moving this logic to inside the CLI in the dedicated automated test selection command. So we won't need `jq` anymore when the time to support alpine comes.